contribute-docs/style-guide/word-choice.md

```diff
@@ -0,0 +1,66 @@
+---
+navigation_title: Word choice
+description: Recommendations for choosing the right words in your documentation.
+---
+
+# Word choice
```

Comment:
Is this in sync with Vale? I'd link to Vale here too and explain that you don't need to commit this to memory if you're using it